core/app/[locale]/(default)/(auth)/layout.tsx

```diff
@@ -1,17 +1,17 @@
 import { PropsWithChildren } from 'react';
 
-import { auth } from '~/auth';
+import { isLoggedIn } from '~/auth';
 import { redirect } from '~/i18n/routing';
 
 interface Props extends PropsWithChildren {
   params: Promise<{ locale: string }>;
 }
 
 export default async function Layout({ children, params }: Props) {
-  const session = await auth();
+  const loggedIn = await isLoggedIn();
   const { locale } = await params;
 
-  if (session) {
+  if (loggedIn) {
```

Comment:
I know this isn't really in scope for this PR, but I wanted to mention that Next.js docs recommend against authorization in layouts: https://nextjs.org/docs/app/building-your-application/authentication#layouts-and-auth-checks

Reply:
Yeah I've chatted with @jorgemoya about this, but I want us to move it back into middleware (`with-auth.ts`). It has already been causing some issues 😅